seed/lib/merging/merging.py

```diff
@@ -177,8 +269,11 @@ def merge_state(merged_state, state1, state2, can_attrs, default=None):
             setattr(merged_state, attr, attr_value)
             # setattr(merged_state, '{0}_source'.format(attr), attr_source)
 
-    merged_extra_data, merged_extra_data_sources = merge_extra_data(state1, state2, default=default)
-
+    merged_extra_data, merged_extra_data_sources = _merge_extra_data(state1, state2, default=default)
     merged_state.extra_data = merged_extra_data
 
+    # merge measures, scenarios,
+    if isinstance(merged_state, PropertyState):
+        _merge_relationships(merged_state, state1, state2)
+
```

Comment:
Hmm, maybe this line clears up my earlier comments in this file.  Does this merge function work on classes other than PropertyState?

Reply:
no, that is the reason for the PropertyState catch.